<launch>
    <!-- Тип автопилота: 0 - PX4, 1 - AruPilot -->
    <arg name="ap_type" default="1"/>
    <!-- Высота в метрах -->
    <arg name="altitude" default="0.5"/>
    <!-- Сторона квадрата в метрах -->
    <arg name="square_side" default="0.25"/>
    <!-- Точность прилета в точку в метрах -->
    <arg name="tolerance" default="0.1"/>
    <!-- Использовать визуальную одометрию или нет -->
    <arg name="use_vision_odometry" default="True"/>

    <node pkg="autotakeoff" type="square_flight_node.py" name="square_flight_node" output="screen">
        <param name="ap_type" value="$(arg ap_type)" type="int"/>
        <param name="altitude" value="$(arg altitude)" type="double"/>
        <param name="square_side" value="$(arg square_side)" type="double"/>
        <param name="tolerance" value="$(arg tolerance)" type="double"/>
        <param name="use_vision_odometry" value="$(arg use_vision_odometry)" type="bool"/>
    </node>
</launch>
